### PR TITLE
Handle capture kit is none

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds a fix for capture kit being none from LIMS.

**How to install**:
1. install on stage of hasta
1. activate stage: `us`

**Test:**
1. run following command: `cg analysis config stillcougar` 

**Expected outcome**:
cg should not crash with a stacktrace with capture_kits.pop() in it

Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [ ] tests executed by @patrikgrenfeldt 
- [ ] "Merge and deploy" approved by @ingkebil 
Thanks for filling in who performed the code review and the test!

This is patch**version bump** because no functionality was changed.

### CODE REVIEW

File: cg/apps/lims/api.py [hunk 1]
```diff
@@ -178,7 +178,10 @@ def capture_kit(self, lims_id: str) -> str:
             message = f"Capture kit error: {lims_sample.id} | {capture_kits}"
             raise LimsDataError(message)
 
-        return capture_kits.pop() or None
+        if len(capture_kits) == 1:
+            return capture_kits.pop()
+        else:
+            return None
```
Comment (ingkebil) [inline cg/apps/lims/api.py]: Would you return None or rather raise an exception?

Comment (patrikgrenfeldt) [reply cg/apps/lims/api.py]: I was trying to preserve the previous seemingly intended behaviour to keep it patchy

**Comment by patrikgrenfeldt** [comment]
image
👍

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**